cilium-cli/connectivity/builder/pod_to_pod_encryption.go

```diff
@@ -25,12 +25,16 @@ func (t podToPodEncryption) build(ct *check.ConnectivityTest, _ map[string]strin
 		WithCondition(func() bool {
 
 			// for wireguard, we can run the podToPodEncryptionV2 tests if we
-			// are on a post v1.18 cluster
+			// are on a post v1.18 cluster and not using aws-cni chaining
```

Comment:
Could you describe in the commit description the impact of this change? I'm not familiar with (or at least don't remember) the difference between v1 and v2. Do we also need to have an issue with assignee to not forget about reverting this?